frontend/micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/Login/index.js

```diff
@@ -7,19 +7,26 @@ import LoginComponent from "./login";
 const EmployeeLogin = () => {
   const { t } = useTranslation();
   const { path } = useRouteMatch();
+  const [loginConfig, setloginConfig] = useState(defaultLoginConfig);
 
-  const { data: mdmsData } = Digit.Hooks.useCommonMDMS(Digit.ULBService.getStateId(), "commonUiConfig", ["LoginConfig"], {
+  const { data: mdmsData, isLoading } = Digit.Hooks.useCommonMDMS(Digit.ULBService.getStateId(), "commonUiConfig", ["LoginConfig"], {
     select: (data) => {
       return {
-        config: data?.MdmsRes?.['commonUiConfig']?.LoginConfig
+        config: data?.commonUiConfig?.LoginConfig
       };
     },
     retry: false,
     enable: false,
```

Comment:
enable false?
it doesnt make that API call right ?

Reply:
yes it shouldn't but still the API call is going, let me check once